src/app/core/components/info-card/info-card.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="card">
+  <div class="message-container">
+    <p>{{ label }}</p>
+  </div>
+  <div class="counter-container">
+    <img src="assets/images/logo-m.png" alt="Classic Mario logo" />
+    <p>{{ counter }}</p>
```

Comment:
Should it be `<p>`? Or maybe `<span>`?

Reply:
Fix it: 9c3204c8a2f7ea6f152e54eb9d5da92ef6f3f554